src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected char type;
```

Comment:
Nice use of protected, though realistically not necessary for description and isDone.